src/twisted/python/reactor_unittest.py

```diff
@@ -0,0 +1,584 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+from unittest import TestCase
```

Comment:
There is no coverage reported for this file, as we don't run the benchmark tests with coverage.

Maybe we should do a single run  / single iteration of the benchmark tests with coverage.

This can help detect unused code and help keep the benchmark tests clean.

In the future, maybe we should also add mypy to the benchmark tests